From d9e81c7199baa728a84e8881317b3709f0ff4d5a Mon Sep 17 00:00:00 2001
From: Alice Ryhl <aliceryhl@google.com>
Date: Tue, 28 Jul 2026 06:12:35 +0000
Subject: [PATCH 09264/15333] rust_binder: update indentation of failed
 transaction print

To properly take the changes from commit bb66b1a34525 ("rust_binder:
only print failure if error has source") into account, the
binder_debug! statement was moved inside the if {} block, and so there
must be one more level of indentation.

Signed-off-by: Alice Ryhl <aliceryhl@google.com>
Link: https://patch.msgid.link/20260728061236.198267-1-aliceryhl@google.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/android/binder/thread.rs | 34 ++++++++++++++++----------------
 1 file changed, 17 insertions(+), 17 deletions(-)

diff --git a/drivers/android/binder/thread.rs b/drivers/android/binder/thread.rs
index b41a62b864da..18a14aa8a835 100644
--- a/drivers/android/binder/thread.rs
+++ b/drivers/android/binder/thread.rs
@@ -1308,23 +1308,23 @@ fn transaction(self: &Arc<Self>, cmd: u32, reader: &mut UserSliceReader) -> Resu
                             ExtendedError::new(info.debug_id as u32, err.reply, source.to_errno());
                     }
 
-                binder_debug!(
-                    FailedTransaction,
-                    "transaction {} to {}:{} failed {:?}, code {} size {}-{}",
-                    if info.is_reply {
-                        "reply"
-                    } else if info.is_oneway() {
-                        "async"
-                    } else {
-                        "call"
-                    },
-                    info.to_pid,
-                    info.to_tid,
-                    err,
-                    info.code,
-                    info.data_size,
-                    info.offsets_size
-                );
+                    binder_debug!(
+                        FailedTransaction,
+                        "transaction {} to {}:{} failed {:?}, code {} size {}-{}",
+                        if info.is_reply {
+                            "reply"
+                        } else if info.is_oneway() {
+                            "async"
+                        } else {
+                            "call"
+                        },
+                        info.to_pid,
+                        info.to_tid,
+                        err,
+                        info.code,
+                        info.data_size,
+                        info.offsets_size
+                    );
                 }
             }
         }
-- 
2.55.0

